config/routes.rb

```diff
@@ -97,6 +89,10 @@
           controller: "vaccination_records/edit",
           action: "update_date_and_time"
     end
+
+    resource :cohort_list, as: :cohort, only: %i[new create] do
```

Comment:
```suggestion
    resource :cohort_import, as: :cohort, only: %i[new create] do
```

Could we name the controller to `CohortImportController` at the same time to match the renamed model? It's the last step to moving away from the old "cohort lists" naming.

Reply:
done!